rspec-core.gemspec

```diff
@@ -13,6 +13,14 @@ Gem::Specification.new do |s|
   s.summary     = "rspec-core-#{RSpec::Core::Version::STRING}"
   s.description = "BDD for Ruby. RSpec runner and example groups."
 
+  s.metadata = {
+    'bug_tracker_uri'   => 'https://github.com/rspec/rspec-core/issues',
+    'changelog_uri'     => "https://github.com/rspec/rspec-core/blob/v#{s.version}/Changelog.md",
+    'documentation_uri' => 'https://rspec.info/documentation/',
```

Comment:
From https://rubygems.org/gems/rspec-core/versions/3.8.0

For the moment it is http://rubydoc.info/gems/rspec-core/file/README.md

Reply:
It seems a bit strange to link to rubydoc.info given there are API docs published on rspec.info:

http://rspec.info/documentation/3.8/rspec-core/

Reply:
I think it should be changed to our docs, it was likely auto generated before